en-US/Punctuation.xml

```diff
@@ -426,19 +426,43 @@
 	 <section id="quotation-marks">
 		<title>Quotation Marks</title>
 
+<!-- Previous guidance
 		 <para>
 			Commas and periods go inside quotation marks.
 		</para>
  		 <para>
-		Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
-		</para>
-	</section>
+			Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
+		</para> -->
 
-<!-- Not implementing this change for now.  		
 		 <para>
-			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside.
+			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation or part of a programming element that uses quotation marks; if not, they go outside the quotation marks. If a sentence ends with a quotation, use only one punctuation mark to end the sentence, and place it inside the closing quotation mark if it is part of the quotation, or outside if it is not.
```

Comment:
Rather than "In technical documentation, punctuation marks,... go inside..." how about "In technical documentation, place punctuation marks,...inside..."
Also remember to continue with the "one sentence per line" guide to make reviews and edits easier. This is relatively new in the SG so you won't see a lot of it yet.